api/src/main/resources/notebooks.yaml

```diff
@@ -23,10 +20,10 @@ produces:
 tags:
   - name: test
     description: Test API
-  - name: cluster
-    description: Clusters API
-  - name: notebooks
-    description: Notebooks API
+  - name: runtime
+    description: Runtimes API
+  - name: proxy
```

Comment:
Also need to update the Welder tags (228 and 259)

Reply:
Sorry, not following - what should be updated? I could potentially move them to the proxy API tag, if that's what you mean, or I could just rename to welder - but this is perhaps out of scope.

Reply:
Right, moving them to the proxy tag

Reply:
Done